WordPress/src/main/res/drawable/reader_button_comment.xml

```diff
@@ -1,9 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
-
-    <item android:drawable="@drawable/ic_comment_disabled" android:state_enabled="false" />
-    <item android:drawable="@drawable/ic_comment_active" android:state_pressed="true" />
-    <item android:drawable="@drawable/ic_comment_active" android:state_selected="true" />
-    <item android:drawable="@drawable/ic_comment" />
-
+    <item android:drawable="@drawable/reader_comment_active" android:state_pressed="true" />
```

Comment:
I got rid of the disabled state for two reasons: one, it's no longer used; and two, the normal color is already grey.